frontend/src/scenes/settings/project/proxyLogic.ts

```diff
@@ -79,10 +79,10 @@ export const proxyLogic = kea<proxyLogicType>([
         createRecord: {
             defaults: { domain: '' },
             errors: ({ domain }: { domain: string }) => ({
-                domain: !isDomain('http://' + domain)
-                    ? 'Do not include the protocol e.g. https://'
-                    : domain.includes('*')
+                domain: domain.includes('*')
```

Comment:
re-jigged here so that `isDomain` can avoid caring about wildcards

Reply:
@daibhin just to check you're happy with this since i've low context on this code :)

Reply:
Yep, looks like it'll still do what was intended